### PR TITLE
Update IoT Edge apply config message on hostname change

### PR DESCRIPTION
**Pull request by ggjjj**
Whenever is a hostname change, the error message suggest to delete all the containers in the system. Instead the new error message would only query IoT edge containers and give a suggestion to delete them

### CODE REVIEW

**Comment by ggjjj** [comment]
image

File: edgelet/iotedge/src/config/apply.rs [hunk 1]
```diff
@@ -237,8 +237,8 @@ async fn execute_inner(
                 .await
                 .map_err(|err| format!("{:?}", err))?;
             if &hostname != new_hostname && !containers.is_empty() {
-                return Err(format!("Cannot apply config because the hostname in the config {} is different from the previous hostname {}. To update the hostname, run the following command which deletes all modules and reapplies the configuration. Or, revert the hostname change in the config.toml file.
-                    sudo iotedge system stop && sudo docker rm -f $(docker ps -aq) && sudo iotedge config apply.
+                return Err(format!("Cannot apply config because the hostname in the config {} is different from the previous hostname {}. To update the hostname, run the following command which deletes all IoT Edge modules and reapplies the configuration. Or, revert the hostname change in the config.toml file.
+                    sudo iotedge system stop && sudo docker rm -f $(docker ps -aq -f \"label=net.azure-devices.edge.owner=Microsoft.Azure.Devices.Edge.Agent\")) && sudo iotedge config apply.
```
Comment (ggjjj) [inline edgelet/iotedge/src/config/apply.rs:241]: should the `docker ps` be with `sudo` as well?

Comment (brotherneeru) [reply edgelet/iotedge/src/config/apply.rs:241]: yes, I'd think so. Since its running in a subshell, youd need to elevate privileges there as well

**Review by brotherneeru — DISMISSED** [review]
Left one comment about error message, but other than that looks good

**Review by brotherneeru — APPROVED** [review]

**Pull request merged**